credsweeper/app.py

```diff
@@ -55,6 +61,7 @@ def __init__(self,
             ml_batch_size: int value, size of the batch for model inference
             ml_threshold: float or string value to specify threshold for the ml model
             find_by_ext: boolean - files will be reported by extension
+            max_depth: int - how deep container files will be researched
```

Comment:
How about change the word? Research seems too much in this case.

```suggestion
            max_depth: int - how deep container files will be scanned
```